app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -880,7 +874,7 @@ private void startNextFetch() {
     // extremely long method
 
     private void readyToProceed(final CommCareSession session) {
```

Comment:
hm, so both of these receive a session as an argument that we're now ignoring, should they recieve the session wrapper instead?